recipes/nmslib/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "nmslib" %}
+{% set version = "2.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/8b/ae/168a8cecf02531b06140d66bdd6e3f05c3fa668ccb5f77369c37e31009eb/nmslib-2.0.5.tar.gz
```

Comment:
Is it possible to use the canonical url like here:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14

Reply:
Yes has fixed